src/invidious/views/template.ecr

```diff
@@ -153,6 +153,11 @@
                         <span><%= translate(locale, "Current version: ") %> <%= CURRENT_VERSION %>-<%= CURRENT_COMMIT %> @ <%= CURRENT_BRANCH %></span>
                     </div>
                 </div>
+            <% if CONFIG.footer %>
+            <div>
+               <h4 style="text-align: center;"><%= CONFIG.footer %></h4>
```

Comment:
Please consider using a `<div>` with a unique class/id and CSS styling, rather than a sub-title (`<h4>`) element. This is in order to make invidious more accessible to blind users ^^

Reply:
Oh ok sure. To be honest, I couldn't think of a way to make the footer spatious enough. Thanks for the heads-up ^^

Reply:
https://github.com/iv-org/invidious/blob/701f03d5e88a8f3c92575965e39b8f96b59f71f9/src/invidious/views/template.ecr#L108-L112

Do you think we should also change the element in the banner section?